### ISSUE TITLE
Regarding the default hyperparameter of cross room

### ISSUE THREAD

**Issue by JJJinx** [LiuSjia/WiSR#4]
I have some trouble in reproduction with the default hyper-parameter under cross-room-user. The following is the hyper-parameter and result I got:
config = [
{
                'data_dir':'/data/',
                'dataset':'CSI',
                'csidataset':'CSIDA',
                'algorithm':'WiSR',
                'seed':0,
                'steps':100,
                'checkpoint_freq':1,
                'results_file':"test_results.txt",
                'evalmode':"fast",
                'debug':True,
                'model_save':200,
                'data_type':'amp',
                'source_domains':None,
                'target_domains':None,
                'batch_size':32,
                'backbone':'CSIResNet',
             },

]

result:
Environment:
      Python: 3.8.6
      PyTorch: 1.8.0+cu111
      Torchvision: 0.9.0+cu111
      CUDA: 11.1
      CUDNN: 8005
      NumPy: 1.23.1
      PIL: 10.1.0
Args:
      algorithm: WiSR
      backbone: CSIResNet
      batch_size: 32
      checkpoint_freq: 1
      csidataset: CSIDA
      data_type: amp
      dataset: CSI
      debug: True
      evalmode: fast
      model_save: 200
      results_file: test_results.txt
      seed: 0
      source_domains: ['room_1_user_4_loc_1']
      steps: 100
      target_domains: ['room_0_user_4_loc_1']
Dataset:
      [CSI] #envs=['room_1_user_4_loc_1'], #classes=6
[36mINFO[0m 11/22 11:17:13 | Target test envs = ['room_0_user_4_loc_1']
[36mINFO[0m 11/22 11:17:13 | n_steps = 100
[36mINFO[0m 11/22 11:17:13 | checkpoint_freq = 1
[36mINFO[0m 11/22 11:17:13 | n_steps is updated to 100 => 101 for checkpointing
[36mINFO[0m 11/22 11:17:13 |
[36mINFO[0m 11/22 11:18:05 | Batch size: 32
Building F
[36mINFO[0m 11/22 11:18:07 | # of params = 3757869
[36mINFO[0m 11/22 11:18:07 | Number of steps per epoch: 35
[36mINFO[0m 11/22 11:18:07 | loss_adv    loss_g      loss_s      step        step_time   target_env
[36mINFO[0m 11/22 11:18:07 | 0.000000    1.852837    0.000000    0           0.199675    0.187500  
[36mINFO[0m 11/22 11:18:07 | 0.000000    1.911585    0.000000    1           0.164915    0.203125  
[36mINFO[0m 11/22 11:18:08 | 0.000000    1.724287    0.000000    2           0.162853    0.187500  
[36mINFO[0m 11/22 11:18:08 | 0.000000    1.811494    0.000000    3           0.164098    0.093750  
[36mINFO[0m 11/22 11:18:08 | 0.000000    1.901460    0.000000    4           0.162116    0.140625  
[36mINFO[0m 11/22 11:18:08 | 0.000000    1.809016    0.000000    5           0.165073    0.078125  
[36mINFO[0m 11/22 11:18:09 | 0.000000    2.062121    0.000000    6           0.165060    0.140625  
[36mINFO[0m 11/22 11:18:09 | 0.000000    2.051316    0.000000    7           0.165567    0.125000  
[36mINFO[0m 11/22 11:18:09 | 0.000000    2.150354    0.000000    8           0.164424    0.187500  
[36mINFO[0m 11/22 11:18:09 | 0.000000    2.164852    0.000000    9           0.164413    0.171875  
[36mINFO[0m 11/22 11:18:10 | 0.000000    2.444517    0.000000    10          0.163322    0.234375  
[36mINFO[0m 11/22 11:18:10 | 0.000000    2.186350    0.000000    11          0.162296    0.187500  
[36mINFO[0m 11/22 11:18:10 | 0.000000    1.817209    0.000000    12          0.162404    0.203125  
[36mINFO[0m 11/22 11:18:10 | 0.000000    1.817271    0.000000    13          0.165377    0.171875  
[36mINFO[0m 11/22 11:18:11 | 0.000000    1.741596    0.000000    14          0.161644    0.203125  
[36mINFO[0m 11/22 11:18:11 | 0.000000    2.646982    0.000000    15          0.163107    0.234375  
[36mINFO[0m 11/22 11:18:11 | 0.000000    2.376044    0.000000    16          0.162694    0.187500  
[36mINFO[0m 11/22 11:18:11 | 0.000000    1.798469    0.000000    17          0.162703    0.156250  
[36mINFO[0m 11/22 11:18:12 | 0.000000    1.900736    0.000000    18          0.161041    0.140625  
[36mINFO[0m 11/22 11:18:12 | 0.000000    1.720469    0.000000    19          0.161801    0.109375  
[36mINFO[0m 11/22 11:18:12 | 0.000000    2.302477    0.000000    20          0.163096    0.171875  
[36mINFO[0m 11/22 11:18:12 | 0.000000    2.721552    0.000000    21          0.165562    0.046875  
[36mINFO[0m 11/22 11:18:13 | 0.000000    2.304985    0.000000    22          0.163722    0.125000  
[36mINFO[0m 11/22 11:18:13 | 0.000000    1.750773    0.000000    23          0.163588    0.156250  
[36mINFO[0m 11/22 11:18:13 | 0.000000    2.090659    0.000000    24          0.163675    0.062500  
[36mINFO[0m 11/22 11:18:14 | 0.000000    1.714972    0.000000    25          0.163398    0.109375  
[36mINFO[0m 11/22 11:18:14 | 0.000000    2.114073    0.000000    26          0.162098    0.236842  
[36mINFO[0m 11/22 11:18:14 | 0.000000    2.880713    0.000000    27          0.162966    0.093750  
[36mINFO[0m 11/22 11:18:14 | 0.000000    1.906345    0.000000    28          0.166236    0.171875  
[36mINFO[0m 11/22 11:18:15 | 0.000000    2.445611    0.000000    29          0.167252    0.125000  
[36mINFO[0m 11/22 11:18:15 | 0.000000    2.400885    0.000000    30          0.163643    0.187500  
[36mINFO[0m 11/22 11:18:15 | 0.000000    1.988564    0.000000    31          0.162891    0.187500  
[36mINFO[0m 11/22 11:18:15 | 0.000000    2.575730    0.000000    32          0.164127    0.218750  
[36mINFO[0m 11/22 11:18:16 | 0.000000    1.869569    0.000000    33          0.163418    0.218750  
[36mINFO[0m 11/22 11:18:16 | 0.000000    1.631584    0.000000    34          0.161520    0.171875  
[36mINFO[0m 11/22 11:18:16 | 0.000000    1.875025    0.000000    35          0.161186    0.125000  
[36mINFO[0m 11/22 11:18:16 | 0.000000    1.611296    0.000000    36          0.161906    0.156250  
[36mINFO[0m 11/22 11:18:17 | 0.000000    1.850182    0.000000    37          0.166049    0.109375  
[36mINFO[0m 11/22 11:18:17 | 0.000000    2.071436    0.000000    38          0.166491    0.093750  
[36mINFO[0m 11/22 11:18:17 | 0.000000    1.730989    0.000000    39          0.164374    0.187500  
[36mINFO[0m 11/22 11:18:17 | 0.000000    1.825742    0.000000    40          0.165700    0.171875  
[36mINFO[0m 11/22 11:18:18 | 0.000000    2.026299    0.000000    41          0.164142    0.171875  
[36mINFO[0m 11/22 11:18:18 | 0.000000    2.000900    0.000000    42          0.162634    0.203125  
[36mINFO[0m 11/22 11:18:18 | 0.000000    1.835025    0.000000    43          0.161602    0.187500  
[36mINFO[0m 11/22 11:18:18 | 0.000000    2.132000    0.000000    44          0.163923    0.171875  
[36mINFO[0m 11/22 11:18:19 | 0.000000    1.728921    0.000000    45          0.164888    0.156250  
[36mINFO[0m 11/22 11:18:19 | 0.000000    1.746263    0.000000    46          0.164520    0.171875  
[36mINFO[0m 11/22 11:18:19 | 0.000000    1.806511    0.000000    47          0.163853    0.093750  
[36mINFO[0m 11/22 11:18:19 | 0.000000    2.397817    0.000000    48          0.164822    0.156250  
[36mINFO[0m 11/22 11:18:20 | 0.000000    2.265171    0.000000    49          0.163827    0.156250  
[36mINFO[0m 11/22 11:18:20 | 0.000000    2.029610    0.000000    50          0.162530    0.218750  
[36mINFO[0m 11/22 11:18:20 | 0.000000    2.039126    0.000000    51          0.164556    0.218750  
[36mINFO[0m 11/22 11:18:21 | 0.000000    2.093800    0.000000    52          0.165652    0.156250  
[36mINFO[0m 11/22 11:18:21 | 0.000000    1.831885    0.000000    53          0.164859    0.052632  
[36mINFO[0m 11/22 11:18:21 | 0.000000    2.042741    0.000000    54          0.164611    0.218750  
[36mINFO[0m 11/22 11:18:21 | 0.000000    2.328248    0.000000    55          0.164239    0.140625  
[36mINFO[0m 11/22 11:18:22 | 0.000000    1.704021    0.000000    56          0.165119    0.171875  
[36mINFO[0m 11/22 11:18:22 | 0.000000    1.838260    0.000000    57          0.166790    0.125000  
[36mINFO[0m 11/22 11:18:22 | 0.000000    1.857275    0.000000    58          0.164948    0.171875  
[36mINFO[0m 11/22 11:18:22 | 0.000000    1.814426    0.000000    59          0.163742    0.187500  
[36mINFO[0m 11/22 11:18:23 | 0.000000    2.176340    0.000000    60          0.165620    0.125000  
[36mINFO[0m 11/22 11:18:23 | 0.000000    1.930907    0.000000    61          0.168762    0.093750  
[36mINFO[0m 11/22 11:18:23 | 0.000000    2.005948    0.000000    62          0.165612    0.125000  
[36mINFO[0m 11/22 11:18:23 | 0.000000    1.953698    0.000000    63          0.165310    0.296875  
[36mINFO[0m 11/22 11:18:24 | 0.000000    1.800272    0.000000    64          0.165021    0.109375  
[36mINFO[0m 11/22 11:18:24 | 0.000000    1.781323    0.000000    65          0.162890    0.187500  
[36mINFO[0m 11/22 11:18:24 | 0.000000    1.872981    0.000000    66          0.163770    0.140625  
[36mINFO[0m 11/22 11:18:24 | 0.000000    1.788383    0.000000    67          0.164109    0.187500  
[36mINFO[0m 11/22 11:18:25 | 0.000000    1.932111    0.000000    68          0.165271    0.109375  
[36mINFO[0m 11/22 11:18:25 | 0.000000    1.764887    0.000000    69          0.166220    0.187500  
[36mINFO[0m 11/22 11:18:25 | 0.000000    1.700549    0.000000    70          0.165979    0.281250  
[36mINFO[0m 11/22 11:18:25 | 0.000000    1.711989    0.000000    71          0.165352    0.093750  
[36mINFO[0m 11/22 11:18:26 | 0.000000    1.966042    0.000000    72          0.165431    0.187500  
[36mINFO[0m 11/22 11:18:26 | 0.000000    1.910140    0.000000    73          0.165005    0.125000  
[36mINFO[0m 11/22 11:18:26 | 0.000000    1.780037    0.000000    74          0.165606    0.140625  
[36mINFO[0m 11/22 11:18:27 | 0.000000    1.758906    0.000000    75          0.165010    0.156250  
[36mINFO[0m 11/22 11:18:27 | 0.000000    1.883132    0.000000    76          0.165592    0.171875  
[36mINFO[0m 11/22 11:18:27 | 0.000000    1.782942    0.000000    77          0.167381    0.109375  
[36mINFO[0m 11/22 11:18:27 | 0.000000    1.615992    0.000000    78          0.163139    0.187500  
[36mINFO[0m 11/22 11:18:28 | 0.000000    1.800573    0.000000    79          0.166363    0.156250  
[36mINFO[0m 11/22 11:18:28 | 0.000000    1.829053    0.000000    80          0.166073    0.131579  
[36mINFO[0m 11/22 11:18:28 | 0.000000    1.691839    0.000000    81          0.164449    0.109375  
[36mINFO[0m 11/22 11:18:28 | 0.000000    1.714615    0.000000    82          0.162752    0.218750  
[36mINFO[0m 11/22 11:18:29 | 0.000000    1.679975    0.000000    83          0.164562    0.156250  
[36mINFO[0m 11/22 11:18:29 | 0.000000    1.758691    0.000000    84          0.164953    0.109375  
[36mINFO[0m 11/22 11:18:29 | 0.000000    1.573097    0.000000    85          0.165778    0.140625  
[36mINFO[0m 11/22 11:18:29 | 0.000000    1.837946    0.000000    86          0.166771    0.171875  
[36mINFO[0m 11/22 11:18:30 | 0.000000    1.583823    0.000000    87          0.167505    0.187500  
[36mINFO[0m 11/22 11:18:30 | 0.000000    1.736017    0.000000    88          0.166155    0.187500  
[36mINFO[0m 11/22 11:18:30 | 0.000000    1.927120    0.000000    89          0.165105    0.140625  
[36mINFO[0m 11/22 11:18:30 | 0.000000    1.692699    0.000000    90          0.164155    0.125000  
[36mINFO[0m 11/22 11:18:31 | 0.000000    1.604398    0.000000    91          0.163895    0.156250  
[36mINFO[0m 11/22 11:18:31 | 0.000000    1.580736    0.000000    92          0.163870    0.125000  
[36mINFO[0m 11/22 11:18:31 | 0.000000    1.569899    0.000000    93          0.167990    0.140625  
[36mINFO[0m 11/22 11:18:31 | 0.000000    1.617810    0.000000    94          0.165309    0.171875  
[36mINFO[0m 11/22 11:18:32 | 0.000000    1.774179    0.000000    95          0.165851    0.078125  
[36mINFO[0m 11/22 11:18:32 | 0.000000    1.423297    0.000000    96          0.165665    0.140625  
[36mINFO[0m 11/22 11:18:32 | 0.000000    1.587312    0.000000    97          0.165509    0.125000  
[36mINFO[0m 11/22 11:18:33 | 0.000000    1.743228    0.000000    98          0.165167    0.093750  
[36mINFO[0m 11/22 11:18:33 | 0.000000    1.599768    0.000000    99          0.164374    0.109375  
[36mINFO[0m 11/22 11:18:33 | 0.000000    1.611295    0.000000    100         0.166181    0.187500  

I have several questions:
1.what is the hyper-parameter you used in the paper.
2.how to set the hyper-para to do the experiment of cross-room only instead of cross-room-user
Many thanks!
